src/linkml_map/cli/cli.py

```diff
@@ -132,7 +129,8 @@ def compile(
     elif target == "markdown":
         compiler = MarkdownCompiler(**compiler_args)
     else:
-        raise NotImplementedError(f"Compiler {target} not implemented")
+        msg = f"Compiler {target} not implemented"
+        raise NotImplementedError(msg)
```

Comment:
create the error message as a variable and then pass it to the error object

see https://docs.astral.sh/ruff/rules/#flake8-errmsg-em